lib/plugins.js

```diff
@@ -2,8 +2,11 @@ var fs = require('fs'),
     inherits = require('util').inherits,
     path = require('path');
 
-var CWD = path.join(__dirname, '../plugins'),
-    THROW_CONFLICTS = true;
+var THROW_CONFLICTS = true;
+
+function getPluginPath(config) {
```

Comment:
It's used in lib/plugins.js now, but not in test/plugins.js; I can modify the test file as well, if you like.